documentation/docs/steps/neoDeploy.md

```diff
@@ -20,6 +14,8 @@ Note that a version is formed by `major.minor.patch`, and a version is compatibl
 
 * **Java 8 or compatible version** - needed by the *Neo-Java-Web-SDK*. Java environment needs to be properly configured (JAVA_HOME, java exectutable contained in path).
 
+## ${docGenParameters}
+
 ## Parameters when using MTA deployment method (default - MTA)
```

Comment:
Should we really keep this tables here (... still handcrafted) when we have the generated table also?

Reply:
Done. Maybe would be also good to generate it from withMandatoryParameter(), but it should be done in another PR.